Xamarin.PropertyEditing/ViewModels/PropertiesViewModel.cs

```diff
@@ -164,6 +204,38 @@ private PropertyViewModel GetViewModel (IPropertyInfo property)
 			return new StringPropertyViewModel (property, this.editors);
 		}
 
+		void FilterData ()
+		{
+			this.properties.Clear ();
+
+			UpdateProperties ();
```

Comment:
The `Properties` list should expose a second, filtered list of the properties, but we can build a filtered list without rebuilding the original property list as `UpdateProperties` does.